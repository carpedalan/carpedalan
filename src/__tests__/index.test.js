import React from 'react';
import { create } from 'react-test-renderer';

import Root from '..';

jest.mock('superagent', () =>
  jest.fn().mockReturnValue({ body: { data: 'data' } }),
);

describe('test', () => {
  it('should do some stuff', () => {
    const instance = create(<Root user="write" />);
    expect(instance.toJSON()).toMatchSnapshot();
  });
});
