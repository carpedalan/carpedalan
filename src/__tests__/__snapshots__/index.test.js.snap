// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test should do some stuff 1`] = `
Array [
  .c0 {
  font-family: montserratregular;
  font-size: 18px;
  text-transform: uppercase;
  -webkit-letter-spacing: 5px;
  -moz-letter-spacing: 5px;
  -ms-letter-spacing: 5px;
  letter-spacing: 5px;
  background: none;
  color: inherit;
  border: none;
  padding: 0;
  cursor: pointer;
  outline: inherit;
  position: fixed;
  z-index: 1;
  top: 0;
  left: 25px;
  background: white;
  padding: 5px;
  box-shadow: rgba(255,255,255) 0px 0px 10px 10px;
  border-radius: 40%;
}

<h1
    className="c0"
    data-test="menu"
    onClick={[Function]}
    size="small"
    type="button"
  >
    Menu
  </h1>,
  .c0 {
  position: fixed;
  width: 200px;
  top: 0;
  left: -20em;
  z-index: 1;
  background: rgba(0,255,231,0.8);
  height: 100%;
  overflow: scroll;
  -webkit-transition: left 200ms ease-in;
  transition: left 200ms ease-in;
  font-family: montserratregular;
  padding: 0em;
}

.c2 {
  margin-left: 0;
  padding-left: 0;
}

.c3 {
  list-style: none;
  margin-left: 0;
  margin-bottom: 1em;
}

.c4 {
  -webkit-text-decoration: none;
  text-decoration: none;
  color: black;
}

.c1 {
  font-family: montserratregular;
  font-size: 24px;
  text-transform: uppercase;
  -webkit-letter-spacing: 5px;
  -moz-letter-spacing: 5px;
  -ms-letter-spacing: 5px;
  letter-spacing: 5px;
  background: none;
  color: inherit;
  border: none;
  padding: 0;
  cursor: pointer;
  outline: inherit;
}

<div
    className="c0"
  >
    <button
      className="c1"
      onClick={[Function]}
      type="button"
    >
      Close âœ–
    </button>
    <ul
      className="c2"
    >
      <li
        className="c3"
      >
        <a
          className="c4"
          href="/admin"
          onClick={[Function]}
        >
          ADMIN
        </a>
      </li>
      <li
        className="c3"
      >
        <a
          className="c4"
          href="/"
          onClick={[Function]}
        >
          HOME
        </a>
      </li>
      <li
        className="c3"
      >
        <a
          className="c4"
          href="/archive"
          onClick={[Function]}
        >
          ARCHIVE
        </a>
      </li>
    </ul>
    <button
      onClick={[Function]}
      type="button"
    >
      toggle themeaa
    </button>
    <button
      data-test="logout"
      onClick={[Function]}
      type="button"
    >
      log out
    </button>
  </div>,
  .c1 {
  font-family: montserratregular;
  font-size: 42px;
  text-transform: uppercase;
  -webkit-letter-spacing: 5px;
  -moz-letter-spacing: 5px;
  -ms-letter-spacing: 5px;
  letter-spacing: 5px;
  text-align: center;
}

.c0 {
  max-width: 67em;
  margin: 0 auto;
}

<section
    className="c0"
  >
    <h1
      className="c1"
      size="large"
    >
      Carpe Dalan
    </h1>
    <div
      style={
        Object {
          "overflow": "visible",
          "width": 0,
        }
      }
    >
      <div>
        <div
          aria-label="grid"
          aria-readonly={true}
          className="ReactVirtualized__Grid ReactVirtualized__List"
          onScroll={[Function]}
          role="grid"
          style={
            Object {
              "WebkitOverflowScrolling": "touch",
              "boxSizing": "border-box",
              "direction": "ltr",
              "height": "auto",
              "overflowX": "hidden",
              "overflowY": "hidden",
              "position": "relative",
              "width": 0,
              "willChange": "transform",
            }
          }
          tabIndex={0}
        />
      </div>
    </div>
  </section>,
]
`;
